krun/vm_defs.py

```diff
@@ -33,6 +33,9 @@
 if DASH is None:
     fatal("dash shell not found")
 
+INSTRUMENTATION_END_PROC_ITER_PREFIX = "@@@ END_IN_PROC_ITER:"
```

Comment:
In the latest batch of changes, I've been shortening to "inst". Are you ok with `INST_END_PROC_ITER_PREFIX` for this one?